lib/comgr/include/amd_comgr.h

```diff
@@ -75,8 +75,15 @@
 #endif
 #endif
 
+#ifndef AMD_COMGR_INTERFACE_VERSION_MAJOR
 #define AMD_COMGR_INTERFACE_VERSION_MAJOR 1
+#endif
+#ifndef AMD_COMGR_INTERFACE_VERSION_MINOR
 #define AMD_COMGR_INTERFACE_VERSION_MINOR 6
+#endif
```

Comment:
I think it may make more sense to just delete these definitions entirely. If we want the build system to always provide them then I don't see much value in having a default of "1.6". Any non-CMake builds of Comgr will have to be updated, but at least it will be an obvious error at compile-time, rather than silently returning bogus version information.

Reply:
Yes, for non cmake builds error will be shown. I will remove this code part and update the commit.

Reply:
PROJECT_VERSION_MAJOR, MINOR is 1 and 6, which could be 1 place where it is maintained and any change should be done to this variable.